test/test_tags.rb

```diff
@@ -7,21 +7,22 @@ def setup
     FileUtils.mkdir_p("tmp")
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
Rather than disabling the AbcSize warning here, maybe we can extra this into a separate test helper class? what do you think?

Reply:
A separate helper class sounds good to me. :+1:
I shall open a separate pull request implementing it.